module/behaviour/strategy/KeyboardWalk/src/KeyboardWalk.hpp

```diff
@@ -23,6 +23,7 @@
 #include <Eigen/Core>
 #include <mutex>
 #include <ncurses.h>
+#undef OK  // ncurses defines OK. We don't need it
```

Comment:
Should wrap this in `// clang-format off` just to maie sure clang-format never does something weird with this

Reply:
I've moved it above the undef - this seemed simpler than adding the boilerplate for the formatting